configure.ac

```diff
@@ -374,7 +379,8 @@ case "$ac_cv_search_crypt_gensalt_rn" in
 			-l*) LIBCRYPT="$ac_cv_search_crypt" ;;
 		esac ;;
 	esac ;;
-esac
+  esac
+])
 AC_CHECK_FUNCS([crypt_r])
```

Comment:
While `AC_SEARCH_LIBS([crypt_gensalt_rn],[crypt])` sets `LIBS`, `LIBCRYPT="$CRYPT_LIBS"` doesn't, so `AC_CHECK_FUNCS([crypt_r])` fails to find `crypt_r` in -lcrypt.

Reply:
Good catch, I updated the PR.